iml-agent/Cargo.toml

```diff
@@ -21,12 +21,12 @@ spinners = "1.0.0"
 libc = "0.2.58"
 log = "0.4.6"
 prettytable-rs = "0.8"
-reqwest = { version = "0.9.17", features = ["default-tls", "tls"] }
+reqwest = { version = "0.9.18", features = ["default-tls", "tls"] }
 hyper = "0.12"
 http = "0.1"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
-structopt = "0.2.17"
+structopt = "0.2.16"
```

Comment:
Is this supposed to be backwards?

Reply:
0.2.17 has been yanked: https://crates.io/crates/structopt/0.2.17